web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
```

Comment:
I can see what we're trying to do with all this importing module names, but it is making the code a lot more complex than possibly necessary?

It could be a lot simpler to just hardcode these? I'm not sure. This is sort of related to @henrahmagix comment below